.github/workflows/maven.yml

```diff
@@ -53,39 +67,43 @@ jobs:
         id: login-ecr
       # Build and push docker image to container repo
       - run: |
-          docker build -t $REGISTRY/$REPOSITORY:$IMAGE_TAG .
-          docker push $REGISTRY/$REPOSITORY:$IMAGE_TAG
+          docker build -t $REGISTRY/$REPOSITORY:${{ env.IMAGE_TAG }} .
+          docker push $REGISTRY/$REPOSITORY:${{ env.IMAGE_TAG }}
         env:
           REGISTRY: ${{ steps.login-ecr.outputs.registry }}
           REPOSITORY: ${{ vars.ECR_REPOSITORY }}
-          IMAGE_TAG: ${{ github.sha }}
+          IMAGE_TAG: ${{ needs.define-image-tag.outputs.tag_name }}
       - name: delete artifact
         uses: geekyeggo/delete-artifact@v2
         with:
           name: saml-jar
 
-#  deploy-dev:
-#    runs-on: ubuntu-latest
-#    environment: development
-#    needs: ecr
-#    env:
-#      KUBE_NAMESPACE: ${{ secrets.KUBE_NAMESPACE_DEV }}
-#    steps:
-#      - name: Checkout
-#        uses: actions/checkout@v3
-#      - name: Update image tag
-#        env:
-#          ECR_URL: ${{ secrets.ECR_URL_DEV }}
-#        run: export IMAGE_TAG=${{ github.sha }} && cat kubectl_deploy/deploy.tpl | envsubst > kubectl_deploy/deployment.yaml
-#      - name: Authenticate to the cluster
-#        env:
-#          KUBE_CLUSTER: ${{ secrets.KUBE_CLUSTER_DEV }}
-#        run: |
-#          echo "${{ secrets.KUBE_CERT_DEV }}" > ca.crt
-#          kubectl config set-cluster ${KUBE_CLUSTER} --certificate-authority=./ca.crt --server=https://${KUBE_CLUSTER}
-#          kubectl config set-credentials deploy-user --token=${{ secrets.KUBE_TOKEN_DEV }}
-#          kubectl config set-context ${KUBE_CLUSTER} --cluster=${KUBE_CLUSTER} --user=deploy-user --namespace=${{ secrets.KUBE_NAMESPACE_DEV}}
-#          kubectl config use-context ${KUBE_CLUSTER}
-#      - name: Apply the updated manifest
-#        run: kubectl -n ${KUBE_NAMESPACE} apply -f kubectl_deploy --namespace=laa-ccms-civil
+  update-helm-chart:
+    if: ${{ github.ref_name == 'development' }}
```

Comment:
Are we not checking the git ref for the app repo here?

Shouldnt this be main? but if you change the actions 'on build ' step this isnt needed